ig/input/fsh/PopulationRecommendation.fsh

```diff
@@ -3,7 +3,8 @@ Parent: Basic
 Id: vp-population-recommendation
 Title: "PopulationRecommendation"
 Description: ""
```

Comment:
Can we add the MS flag to `code` so it shows up in the must-support snapshot table? It's already set to 1..1 by default.

Reply:
Addressed at da97313